crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -47,6 +47,55 @@ pub enum Visibility {
     Visible,
 }
 
+impl Visibility {
+    /// Returns `Visibility::Inherited` if true, else returns `Visibility::Hidden`.
+    #[inline]
+    pub fn from_bool_inherited(value: bool) -> Self {
+        if value {
+            Visibility::Inherited
+        } else {
+            Visibility::Hidden
+        }
+    }
+    /// Returns `true` if `self` is `Visibility::Visible` or `Visibility::Inherited`, else returns `false`.
+    #[inline]
+    pub fn to_bool_inherited(self) -> bool {
```

Comment:
I'm not sure about the naming scheme. `to_bool_inherited` doesn't feel very intuitive, `visibility != Visibility::Hidden` seems much clearer.